openshift/tests-extension/hack/run-tests-clean.sh

```
@@ -0,0 +1,221 @@
+else
+    echo -e "${RED}${BOLD}✗ ${FAILED_FINAL} TEST(S) FAILED${RESET}"
+    echo ""
+
+    # Extract and display failed test details
+    echo -e "${CYAN}${BOLD}════════════════════════════════════════════════════════${RESET}"
+    echo -e "${CYAN}${BOLD}  Failed Test Details${RESET}"
+    echo -e "${CYAN}${BOLD}════════════════════════════════════════════════════════${RESET}"
+    echo ""
+
+    # Parse JSON output for failed tests with details
+    grep '"result": "failed"' $TMPFILE -A 10 | while IFS= read -r line; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)